tests/test_examples.py

```diff
@@ -8,57 +8,62 @@
 
 from tests.common import EXAMPLES_DIR, requires_cuda
 
-CPU_EXAMPLES = []
-CUDA_EXAMPLES = []
+CPU_EXAMPLES = [
+    ('air/main.py', ['--num-steps=1']),
+    ('bayesian_regression.py', ['--num-epochs=1']),
+    ('contrib/gp/sv-dkl.py', ['--epochs=1', '--num-inducing=4', '--no-cuda']),
+    ('contrib/named/mixture.py', ['--num-epochs=1']),
+    ('contrib/named/tree_data.py', ['--num-epochs=1']),
+    ('dmm/dmm.py', ['--num-epochs=1']),
+    ('dmm/dmm.py', ['--num-epochs=1', '--num-iafs=1']),
+    ('inclined_plane.py', ['--num-samples=1']),
+    ('schelling.py', ['--num-samples=1']),
+    ('schelling_false.py', ['--num-samples=1']),
+    ('vae/ss_vae_M2.py', ['--num-epochs=1']),
+    ('vae/ss_vae_M2.py', ['--num-epochs=1', '--aux-loss']),
+    ('vae/ss_vae_M2.py', ['--num-epochs=1', '--enum-discrete=parallel']),
+    ('vae/ss_vae_M2.py', ['--num-epochs=1', '--enum-discrete=sequential']),
+    ('vae/vae.py', ['--num-epochs=1']),
+    ('vae/vae_comparison.py', ['--num-epochs=1']),
```

Comment:
this is definitely more readable but seems less scalable? most examples will only need `num-epochs` set, but will require a manual modification of this file each time

Reply:
There are examples that need multiple parameters like `ss_vae` and require us to specify circuitous logic which seems very error prone. e.g. `if 'aux_loss' in text`. 

We just need one entry here at the time when you write an example, with no future modification needed. And if you forget it, the coverage test will fail. So w.r.t. scalability, I think that our bottleneck to writing more examples will just be the effort required in writing one, rather than adding one entry here! 😄